projects/exui-common-lib/src/lib/components/generic-filter/generic-filter-utils.ts

```diff
@@ -14,7 +14,7 @@ export function maxSelectedValidator<T>(max: number): ValidatorFn {
   };
 }
 
-export function getCheckBoxesValues(options: { key: string, label: string, selectAll?: true }[], values: any): any[] {
+export function getValues(options: { key: string, label: string, selectAll?: true }[], values: any[]): any[] {
```

Comment:
Minor comment - any type used twice here - should we make types for these?

Reply:
I don't think so because the values could contain anything. Boolean, Strings or Objects